ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-path-selection.md

```diff
@@ -122,9 +127,27 @@
           # Load-balance policy name.
         - name: <str; required; unique>
 
+          # Prefer paths with lowest hop-count.
+          hop_count_lower: <bool>
+
+          # Jitter requirement for this load balance policy in milliseconds.
+          jitter: <int; 0-10000>
+
+          # One way delay requirement for this load balance policy in milliseconds.
+          latency: <int; 0-10000>
+
+          # Loss Rate requirement for this load balance policy in milliseconds.
+          loss_rate: <str>
```

Comment:
Maybe add some more detail in description that this can take a float as string.

Reply:
Plus it says millisecond .. good catch :)